release-content/migration-guides/no_material_in_prelude.md

```diff
@@ -0,0 +1,30 @@
+Code that previously relied on `Material` from `prelude` will need to add:
```

Comment:
This needs a heading and linked PR number :) See the sibling migration guides for a template to copy-paste.